shenyu-client/shenyu-client-motan/src/main/java/org/apache/shenyu/client/motan/MotanServiceBeanPostProcessor.java

```diff
@@ -108,6 +110,12 @@ private void handler(final Object bean) {
         if (group == null) {
             group = ((BasicServiceConfigBean) applicationContext.getBean(BASE_SERVICE_CONFIG)).getGroup();
         }
+        if (timeout == null) {
```

Comment:
timeout ，maybe config in props

Reply:
> timeout ，maybe config in props

`timeout` comes from `com.weibo.api.motan.config.AbstractInterfaceConfig#requestTimeout`, it's ok.